x11-misc/arandr/arandr-0.1.10.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_6 python3_7 )
+inherit distutils-r1
+
+DESCRIPTION="Another XRandR GUI"
+HOMEPAGE="https://christian.amsuess.com/tools/arandr/"
+SRC_URI="https://christian.amsuess.com/tools/${PN}/files/${PN}-${PV}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=dev-python/pygobject-3[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	>=dev-python/docutils-0.6[${PYTHON_USEDEP}]
+	x11-apps/xrandr
+"
+
+src_prepare() {
+	local i p
+	# simulate gettext behavior:
+	#  LINGUAS unset => install all
+	#  LINGUAS="" => install none
+	#  LINGUAS="de fr" => install de and fr
+	if [[ -n "${LINGUAS+x}" ]] ; then # if LINGUAS is set
+		for i in $(cd "${S}"/data/po ; for p in *.po ; do echo ${p%.po} ; done) ; do # for every supported language
```

Comment:
Obscenely long line. Move the comment to be on its own line. It's striving too hard to minimize line count, so optionally split this into two steps instead of one.